### PR TITLE
Add ancom

### PR DESCRIPTION
**Pull request by sjanssen2**
Add an analyses function to execute q2's Ancom. Function will draw a graph.

### CODE REVIEW

File: ggmap/analyses.py [hunk 1]
```diff
@@ -4082,6 +4082,144 @@ def post_cache(cache_results):
                      array=1,
                      **executor_args)
 
+def ancom(counts: pd.DataFrame, rank, taxonomy: pd.Series, grouping: pd.Series, min_mean_abundance_per_feature: float=0.005,
+          ppn=1, pmem='4GB', **executor_args):
+    """Execute ANCOM analysis through Qiime2.
+
+    Paramaters
```
Comment (Anna-Rehm) [inline ggmap/analyses.py:4089]: Parameter description for grouping is missing.

File: ggmap/analyses.py [hunk 2]
```diff
@@ -4082,6 +4082,144 @@ def post_cache(cache_results):
...
+    -------
+    """
+
+    COL = 'COLANCOM'
+    # general sanity checks
+    (counts, grouping) = sync_counts_metadata(counts, grouping.dropna())
+    if grouping.value_counts().shape[0] < 2:
+        raise ValueError("Your grouping information has only ONE group. It must be exactly two!")
+    if grouping.value_counts().shape[0] > 2:
+        raise ValueError("Your grouping information more than two groups. It must be exactly two!")
```
Comment (Anna-Rehm) [inline ggmap/analyses.py:4116]: Typo: missing word "has"
```suggestion
        raise ValueError("Your grouping information has more than two groups. It must be exactly two!")
```

File: ggmap/analyses.py (hunk 1 above)
Comment (sjanssen2) [reply ggmap/analyses.py:4089]: added with latest commit

File: ggmap/analyses.py (hunk 2 above)
Comment (sjanssen2) [reply ggmap/analyses.py]: fixed

**Pull request merged**